{{cookiecutter.project_dir}}/.lighthouse/jenkins-x/release.yaml

```diff
@@ -67,9 +67,9 @@ spec:
               script: |
                 #!/usr/bin/env sh
                 source .jx/variables.sh
-                jx promote --app {{cookiecutter.project_dir}}-ui --version $VERSION -b --all --timeout 1h --no-poll
-                jx promote --app {{cookiecutter.project_dir}}-api --version $VERSION -b --all --timeout 1h --no-poll
-                jx promote --app {{cookiecutter.project_dir}}-batch-process --version $VERSION -b --all --timeout 1h --no-poll
+                jx promote --app {{cookiecutter.project_dir}}-api --version $VERSION -b --all --timeout 1h
```

Comment:
why do we remove the --no-poll param?

Reply:
To force apps to be promoted sequentially and avoid conflict as in => https://github.com/Aitomatic/ai-ci-cluster-dev/pull/103/conflicts